src/WebJobs.Script/Binding/BlobBinding.cs

```diff
@@ -70,5 +73,17 @@ public override Collection<CustomAttributeBuilder> GetCustomAttributes()
 
             return attributes;
         }
+
+        private FileAccess GetAttributeAccess(Type parameterType)
+        {
+            // The types bellow only support Read/Write access.
+            // When using them we ignore the acces and always assume ReadWrite
+            if (parameterType == typeof(ICloudBlob) || parameterType == typeof(CloudBlockBlob) || parameterType == typeof(CloudPageBlob))
```

Comment:
CloudBlobDirectory is also a supported binding (but isn't on the PDF doc). Technically that is also a RW binding. That binding works currently however.